### PR TITLE
Temporary fix for Capybara random tests failure

### PR DESCRIPTION
**Pull request by danielkryska**
- repeat test cases 3 times until they pass or return failure

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -76,6 +77,12 @@
 
   # Use mock of middleware
   config.include Devise::Test::ControllerHelpers, type: :controller
+
+  # Re-run tests on failure to prevent random failures
+  config.include RSpec::Repeat
+  config.around :each do |example|
```
Comment (wujuu) [inline spec/rails_helper.rb:83]: Rerunning undeterministic tests and hoping they would pass seems kind of sloppy but I guess it's OK as a quick fix for now

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -76,6 +77,12 @@
...
+    repeat example, 4.times, verbose: true
```
Comment (wujuu) [inline spec/rails_helper.rb:84]: Also, are you running EVERY example 4 times? Not just the capybara ones?

**Review by wujuu — DISMISSED** [review]

File: spec/rails_helper.rb (hunk 1 above)
Comment (danielkryska) [reply spec/rails_helper.rb:83]: As in the title `Temporary fix for Capybara random tests failure`, it's only an ad hoc solution. Although I suppose that the fix is going to stay with us for longer.

File: spec/rails_helper.rb (hunk 2 above)
Comment (danielkryska) [reply spec/rails_helper.rb:84]: It's running strictly 3 times with normal capybara run.

File: spec/rails_helper.rb (hunk 1 above)
Comment (danielkryska) [reply spec/rails_helper.rb:83]: I understand your worries about the made solution, but please notice that we're not living in a perfect world. Running failing test cases multiple times is a common solution of companies, even when those have perfectly working automatic QA pipes.

Comment (michal-szostak) [reply spec/rails_helper.rb:83]: I do agree @wujuu , but we desperately need some kind of fix for this. We'll be working on a better solution in the next sprint.

**Review by michal-szostak — APPROVED** [review]

**Pull request merged**